test/integration/server.cc

```diff
@@ -153,6 +157,9 @@ void IntegrationTestServer::onWorkerListenerRemoved() {
 void IntegrationTestServer::serverReady() {
   pending_listeners_ = server().listenerManager().listeners().size();
   server_set_.setReady();
+  if (on_server_ready_cb_ != nullptr) {
```

Comment:
I think this should be done before the conditional set on the previous line, right?

Reply:
yes, that's right.  Let me change that